### ISSUE TITLE
[DRIVER REQUEST] Adafruit Thermal Printer

### ISSUE THREAD

**Issue by jacobhq** [willswire/balenaPrint#21]
I recently followed [this guide](https://learn.adafruit.com/networked-thermal-printer-using-cups-and-raspberry-pi), and set up the printer on Raspbian. Any ideas on how to get it to work with this project?

I have connected the printer via the txd, rxd and gnd pins on the raspberry pi zero, and it has power. I also have a ppd file with the info for the printer.

**What I'm using**
- Printer - https://www.adafruit.com/product/597
- Raspberry pi zero

Basically, I would like this to be able to detect serial printers.

**Comment by jacobhq** [comment]
This could also be related to lack of drivers, so any help on how to install those would be appreciated.￼

**Comment by willswire** [comment]
Hey @jacobhq! Let me take a look at the drivers required and see if we can add them. It would be a simple matter of installing them in the Docker build steps (like we did in #2).
